master.js

```diff
@@ -446,7 +424,7 @@ $(function() {
                           : 'engines';
 
     // First, hide the platformtype bar if we're sorting by features.
-    $('.platformtype')[(sortByFeatures || sortByFlaggedFeatures) ? 'hide' : 'show']();
+    $('body')[(noPlatformtypeBar || sortByFeatures || sortByFlaggedFeatures) ? 'addClass' : 'removeClass']('hide-platformtype');
```

Comment:
I know we already had it like this but should probably use `toggleClass` here at some point